tests/base_test.go

```diff
@@ -0,0 +1,649 @@
+			}
+			sds, ok := diskResponse.MustDisk().StorageDomains()
+			if !ok {
+				t.Fatal("Failed to get sds")
+			}
+			for _, sd := range sds.Slice() {
+				sdId, ok := sd.Id()
+				if !ok {
+					t.Fatal("Failed to get sd id")
+				}
+				pair := v1beta1.StoragePair{
+					Source: ref.Ref{ID: sdId},
+					Destination: v1beta1.DestinationStorage{
+						StorageClass: sc,
+					},
+				}
+				sdPairs = append(sdPairs, pair)
+			}
+		}
+		nicsResponse, err := vmService.NicsService().List().Send()
+		nics, ok := nicsResponse.Nics()
+		if !ok {
+			t.Fatal("Failed to get nics")
+		}
+		nicPairs = []v1beta1.NetworkPair{}
+		for _, nic := range nics.Slice() {
+			vnicService := ovirtCl.connection.SystemService().VnicProfilesService().ProfileService(nic.MustVnicProfile().MustId())
+			vnicResponse, err := vnicService.Get().Send()
+			if err != nil {
+				t.Fatal(err, "Failed to get vnic service")
+			}
+			profile, ok := vnicResponse.Profile()
+			if !ok {
+				t.Fatal("Failed to get nic profile")
+			}
+
+			network, ok := profile.Network()
+			if !ok {
+				t.Fatal("Failed to get network")
+			}
+			networkId, ok := network.Id()
+			if !ok {
+				t.Fatal("Failed to get network id")
+			}
+			pair := v1beta1.NetworkPair{
+				Source: ref.Ref{ID: networkId},
+				Destination: v1beta1.DestinationNetwork{
+					Type: "pod",
+				},
+			}
+			nicPairs = append(nicPairs, pair)
+		}
+	}
+
+	storageMap := &v1beta1.StorageMap{
+		TypeMeta: v1.TypeMeta{
+			Kind:       "StorageMap",
+			APIVersion: "forklift.konveyor.io/v1beta1",
+		},
+		ObjectMeta: v1.ObjectMeta{
+			Name:      "storage-map-test",
+			Namespace: namespaceName,
+		},
+		Spec: v1beta1.StorageMapSpec{
+			Map: sdPairs,
+			Provider: provider.Pair{
+				Destination: corev1.ObjectReference{
+					Name:      "host",
+					Namespace: localProviderNamespace,
+				},
+				Source: corev1.ObjectReference{
+					Name:      providerIdentifier.Name,
+					Namespace: providerIdentifier.Namespace,
+				}},
+		},
+	}
+
+	log.Info("Creating Storage Map...")
+	err = cl.Create(context.TODO(), storageMap, &client.CreateOptions{})
+	if err != nil {
+		t.Fatal(err, "Failed to create storagemap")
+	}
+
+	networkMap := &v1beta1.NetworkMap{
+		TypeMeta: v1.TypeMeta{
+			Kind:       "NetworkMap",
+			APIVersion: "forklift.konveyor.io/v1beta1",
+		},
+		ObjectMeta: v1.ObjectMeta{
+			Name:      "network-map-test",
+			Namespace: namespaceName,
+		},
+		Spec: v1beta1.NetworkMapSpec{
+			Map: nicPairs,
+			Provider: provider.Pair{
+				Destination: corev1.ObjectReference{
+					Name:      "host",
+					Namespace: localProviderNamespace,
+				},
+				Source: corev1.ObjectReference{
+					Name:      providerIdentifier.Name,
+					Namespace: providerIdentifier.Namespace,
+				}},
+		},
+	}
+
+	log.Info("Creating Network Map...")
+	err = cl.Create(context.TODO(), networkMap, &client.CreateOptions{})
+	if err != nil {
+		t.Fatal(err, "Failed to create networkmap")
+	}
+
+	planCustomResource := &v1beta1.Plan{
```

Comment:
```suggestion
	plan := &v1beta1.Plan{
```
I don't think the CustomResource postfix adds any value here